srcpkgs/libdrm/template

```diff
@@ -3,7 +3,7 @@ pkgname=libdrm
 version=2.4.113
 revision=1
```

Comment:
won't be built unless revbumped
```suggestion
revision=2
```

Reply:
Right, I wasn't sure if adding a new subpackage would have an effect here, since there are two different targets and the contents of the base package do not change. It would still have to build that though, in order to generate contents for the new subpackge.

Reply:
they aren't really targets but split destinations for files, subpackages can only be built if the main package is built